packages/block-library/src/tabs/index.php

```diff
@@ -132,6 +132,35 @@ static function ( array $tab ) use ( &$tab_index ): array {
 	);
 }
 
+/**
+ * Build inline CSS custom properties for border settings.
+ *
+ * @param array $attributes Block attributes.
+ *
+ * @return string Inline CSS string.
+ */
+function block_core_tabs_generate_border_styles( array $attributes ): string {
+	$radius = $attributes['style']['border']['radius'] ?? null;
+
+	if ( empty( $radius ) ) {
```

Comment:
I wonder if this is too strict. Would something like this be better?

```suggestion
	if ( ! isset( $radius ) || $radius === null || $radius === '' ) {
```

Reply:
The only two scenarios not found here are `false` and `0` (or 0) that `empty` cover

If `false` (unlikely case, unless something is going wrong), then its fine to not return anything

And for 0, `--tab-border-radius: 0` and nothing at all, is the same for styling purposes, but we are saving some code. 

So I feel that `empty` is the best option here. I'm not a big fan of defaulting to `empty` also but I feel that here is fine. Also, is very idiomatic for readability (probably why most generally pick it as the first option).

Reply:
Ah I see, I think it's OK to leave it as just checking for `empty` then.